config/ember-try.js

```diff
@@ -1,95 +1,108 @@
-/* eslint-disable */
+/* eslint-disable n/no-unpublished-require */
 'use strict';
 
 const getChannelURL = require('ember-source-channel-url');
 
-module.exports = function () {
-  return Promise.all([
-    getChannelURL('release'),
-    getChannelURL('beta'),
-    getChannelURL('canary'),
-  ]).then((urls) => {
-    return {
-      usePnpm: true,
-      scenarios: [
-        {
-          name: 'ember-lts-3.16',
-          npm: {
-            devDependencies: {
-              'ember-source': '~3.16.0',
-              'ember-qunit': '^5.1.5',
-            },
+module.exports = async function () {
+  return {
+    usePnpm: true,
+    scenarios: [
+      {
+        name: 'ember-lts-3.16',
+        npm: {
+          devDependencies: {
+            '@ember/test-helpers': '^2.4.0',
```

Comment:
also need to add ember-resolver v11.0.1 until ember-source < 4.12

Reply:
@patricklx Yeah, is that safe to do? I honestly don't know if we will use consuming app's resolvers when they boot inspector? If so, that sounds like a good fix.

Reply:
@RobbieTheWagner it's safe. It's only used for the UI. We do not consume it from app code

Reply:
Thanks @patricklx, that fixed some of the tests, but looks like we still have some issues.

Reply:
some are failing because of ember-cli-app-version, I do not think we need it. can just be removed from deps

Reply:
Probably need more cleanup. Ember 6 beta removes support for extend prototypes.